src/js/hca/config/form.js

```diff
@@ -381,6 +384,10 @@ const formConfig = {
                 labels: dischargeTypeLabels
               }
             },
+            'ui:options': {
+              showPrefillMessage: true,
+              prefillMessage: 'military'
```

Comment:
If testing suggests that the additional message is helpful, we may consider using an array to store the pages on which the message can appear, and update the current logic for setting prefillStatust to indicate which information specifically was prefilled.